test/datahike/test/attribute_refs/entity.cljc

```diff
@@ -77,9 +77,6 @@
                                  [{:db/id 1, :name "Ivan"}
                                   {:db/id 2, :name "Oleg"}])
         db (d/db-with ref-db entities)]
-    (is (nil? (d/entity db nil)))
-    (is (nil? (d/entity db "abc")))
-    (is (nil? (d/entity db :keyword)))
```

Comment:
These assertions are here to show that the function is able to tolerate garbage in input - I specced it more precisely, so these lines make the tests fail since the input doesn't conform to the spec. We could change the spec to be `any?` instead, so we are able to accept any input. I don't have a strong opinion on either.